src/gui/guiFormSpecMenu.cpp

```diff
@@ -2715,7 +2715,7 @@ void GUIFormSpecMenu::parseModel(parserData *data, const std::string &element)
 	std::vector<std::string> textures = split(parts[3], ',');
 	std::vector<std::string> vec_rot = split(parts[4], ',');
 	bool inf_rotation = is_yes(parts[5]);
-	bool mousectrl = !is_yes(parts[6]) || parts[6].empty(); // default true
+	bool mousectrl = is_yes(parts[6]) || parts[6].empty(); // default true
```

Comment:
Also, alternatively:
```suggestion
	bool mousectrl = parts[6].empty() ? true : is_yes(parts[6]); // default true
```

Reply:
this has no benefit whatsoever. If you want the performance improvement, you can just do:

```c++
parts[6].empty() || is_yes(parts[6]);
```

Reply:
I don't think such small performance differences matter here.
The Elvis operator would also work here:
```suggestion
	bool mousectrl = parts[6].empty() ?: is_yes(parts[6]); // default true
```

Reply:
The elvis operator is non-standard - only works in GCC. It's also completely unnecessary, they're both booleans